### PR TITLE
Remove duplicate v1.9.x/sof-v1.9.2/intel-signed/*.ldc files

### PR DESCRIPTION
**Pull request by marc-hb**
See commit messages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ sudo ./install.sh v1.N.x/v1.N-rcM
 The go.sh and install.sh for pre-v1.7 releases have been deleted,
 you can find them in the history.
 
+There are no branches anymore, everything is in the default branch.
```
Comment (fredoh9) [inline README.md:13]: You mean,
There is no other branch anymore, everything is in the default(main) branch

Comment (marc-hb) [reply README.md]: Rephrased, thanks.

**Comment by keqiaozhang** [comment]
Confirmed that the `*ldc` files are same.

**Review by keqiaozhang — APPROVED** [review]

**Review by mengdonglin — APPROVED** [review]

**Review by aiChaoSONG — APPROVED** [review]

**Comment by aiChaoSONG** [comment]
Let's merge for the release

**Pull request merged**